docs/MESSAGING.md

```diff
@@ -19,10 +19,30 @@ Open `app/App_Resources/iOS/Info.plist` and add this to the bottom:
 ```xml
 <key>UIBackgroundModes</key>
 <array>
-	<string>remote-notification</string>
+    <string>remote-notification</string>
 </array>
 ```
 
+For Android you will not get the title and body if the notification was received while the application was in background, but you will get the *data* payload.
+Add the following services in the `app/App_Resources/Android/AndroidManifest.xml` to enable advanced FCM messaging:
+```
+<manifest ... >
```

Comment:
Thanks for this! Do you think we'd be able to add this to the postinstall script? Not as part of this PR, just would like to know if you think it would be a wise thing to do.